provider/resource_keycloak_openid_client_permission.go

```diff
@@ -0,0 +1,300 @@
+package provider
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mrparkers/terraform-provider-keycloak/keycloak"
+)
+
+func resourceKeycloakOpenidClientPermission() *schema.Resource {
```

Comment:
Should this not plural resourceKeycloakOpenidClientPermissions instead of resourceKeycloakOpenidClientPermission?

Reply:
Yep, I'll rename this resource.